docs/security/rbac.md

```diff
@@ -366,6 +366,42 @@ You can create additional policies to further limit user access. Use the web UI
     ]
 }
 ```
+## Multiple Resources Statements
+
+Starting from version v1.54.0, (for on-prem users using fluffy > v0.11.0) LakeFS supports specifying multiple resources in a single RBAC statement.
```

Comment:
It's a bit unclear that this configuration applies also to cloud. I think this sentence is too difficult. 
Let's split it up! (Also note it's called lakeFS!)
```suggestion
lakeFS supports specifying multiple resources in a single RBAC statement. This is available on lakeFS Cloud, or on lakeFS Enterprise if starting at version lakeFS v1.54.0 and Fluffy v0.12.0
```
I'm off course unsure about the actual Fluffy version number, I bumped the minor version by 1 because it said ">". It's a good idea to say the first version that works (like you did lakeFS) and not the last version that doesn't work (Fluffy >).